mri/operators/fourier/non_cartesian.py

```diff
@@ -134,13 +134,13 @@ def _adj_op(self, x):
         return np.copy(self.plan.adjoint()) / np.sqrt(self.plan.M)
 
     def adj_op(self, x):
-        """ This method calculates inverse masked non-cartesian Fourier
+        """ This method computes inverse masked non-uniform Fourier
```

Comment:
(D401) First line should be in imperative mood.
(D210) No whitespaces allowed surrounding docstring text
```suggestion
        """Compute inverse masked non-uniform Fourier
```